packages/lib/event-types/transformers/api-response.ts

```diff
@@ -255,9 +258,21 @@ function getResponseEventTypeFutureBookingLimits(
       return undefined;
   }
 }
+
+function getResponseEventTypeRecurrence(
+  transformRecurringEvent: TransformRecurringEventSchema_2024_06_14
+): Recurrence_2024_06_14 {
+  return {
+    interval: transformRecurringEvent.interval,
+    occurrences: transformRecurringEvent.count,
+    frequency: Frequency[transformRecurringEvent.freq] as keyof typeof Frequency,
+  } as unknown as Recurrence_2024_06_14;
```

Comment:
not sure why we need unknown here ? is the TS not working? can't we directly do `satisfies Recurrence_2024_06_14`

Reply:
made changes in https://github.com/calcom/cal.com/pull/16251/commits/126a7b7cbffc50825b3e4ada442db69221d4edd1

Reply:
Fixed